src/parser/FactorTrigramHeadAutomaton.h

```diff
@@ -38,12 +38,12 @@ class FactorTrigramHeadAutomaton : public GenericFactor {
       for (int s = m+1; s <= index_siblings_.size(); ++s) {
         //CHECK_GE(index_siblings_[m][s], 0);
         int index = index_siblings_[m][s];
-	if (index >= 0) {
-	  stream << " " << setprecision(9) << additional_log_potentials_[index];
-	  ++total;
-	} else {
-	  stream << " " << setprecision(9) << 0.0;
-	}	  
+  if (index >= 0) {
```

Comment:
This indentation is messed up. I think it was originally wrong (tabs instead of spaces, my fault) and then you might have ran some script.